### ISSUE TITLE
Outgoing email from a WordPress grain on Sandstorm Oasis never gets sent?

### ISSUE THREAD

**Issue by meitar** [sandstorm-io/sandstorm#1902]
**TL;DR:** Does the Sandstorm-packaged WordPress app have the capability to send outgoing email on Oasis?

Hi! First, Sandstorm seems very promising and I'm really grateful to the devs for making such a user-focused architecture feasible. I work closely with many small community groups for which I think Sandstorm might be an incredible tool and I really want to see it work for them.

My issue, or possibly user confusion, is that **outgoing email originating from within an app on Sandstorm Oasis doesn't get delivered**. More specifically:

* I am testing a WordPress plugin on a new grain in Sandstorm Oasis.
* The plugin itself uses WordPress's built-in `wp_mail()` function (and that, subsequently, uses PHPMailer).
* This plugin's outgoing email works without issue on other servers.
* In Sandstorm, the email never seems to get delivered and there are no errors reported in the Debug Log view.

I'm aware that a grain is purposefully isolated from the outside world and email, in particular, must traverse via Cap'n Proto (as described on your [docs](https://docs.sandstorm.io/en/latest/developing/email-from-apps/)), so I *assume* that the problem is not specific to this plugin, but rather with the WordPress packaged app itself?

What, if anything, must I do to grant my grain the capability to create and send email?

I'm 100% new to Sandstorm, so I apologize if this question makes little sense, or if I'm clearly misunderstanding the use case here. I have gone over what docs I could find as well as skimmed other issues here (like #545 and #1877) but did not glean an answer to the above question from them.

Thanks for your time and for making Sandstorm!

**Comment by meitar** [comment]
Ah, I just found the WordPress-as-a-Sandstorm-app repo. Feel free to close this issue if that tracker is the appropriate place for this issue. Either way, thanks.

**Comment by dwrensha** [comment]
Further discussion: https://github.com/dwrensha/wordpress-sandstorm/issues/12
